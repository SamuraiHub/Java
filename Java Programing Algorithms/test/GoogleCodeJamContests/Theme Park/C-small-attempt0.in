50
4 6 4
1 4 2 1
100 10 1
1
5 5 10
2 4 2 3 4 2 1 2 1 3
1 1 1
1
100 2 2
1 2
1000 100 10
4 3 9 1 2 5 6 9 1 10
1 10 10
4 3 4 1 2 5 6 9 1 10
1 10 9
1 1 1 1 1 1 1 1 1
1000 10 9
1 1 1 1 1 1 1 1 1
934 9 10
8 8 8 8 8 8 8 8 8 8
734 3 10
1 2 3 3 2 1 1 2 3 1
345 19 10
10 10 10 10 10 10 10 10 10 10
765 10 10
10 1 10 1 10 1 10 1 10 1
1000 100 10
10 10 10 10 10 10 10 10 10 10
734 20 10
1 2 3 4 5 6 7 8 9 10
631 16 10
10 9 8 7 6 5 4 3 2 1
2 13 9
8 4 3 5 8 9 6 9 5
4 10 9
8 3 7 2 1 3 8 7 5
4 13 9
4 1 7 7 5 6 3 4 9
3 12 8
6 4 7 10 2 4 1 1
87 11 10
9 9 1 7 6 5 5 2 9 7
477 11 9
5 1 6 4 8 4 4 3 7
391 6 10
3 5 2 4 3 2 6 6 2 6
138 7 9
7 2 4 1 1 3 3 7 3
8 28 10
2 1 10 5 5 1 9 7 3 2
8 27 10
10 5 3 8 7 7 9 9 10 8
8 22 9
10 3 1 8 7 4 8 8 8
6 28 10
10 1 1 1 10 6 10 4 9 10
10 14 10
5 8 8 9 7 5 8 5 7 5
78 27 10
5 6 6 8 5 7 5 6 7 9
271 29 8
9 5 7 6 9 7 6 8
141 49 10
6 6 5 6 9 7 6 9 9 9
310 34 9
5 7 6 7 9 7 9 5 7
556 31 9
5 8 8 9 9 6 5 5 6
102 46 9
5 5 9 9 5 6 7 8 8
360 12 10
9 8 8 6 6 6 7 6 6 5
250 70 8
8 7 9 7 8 8 7 6
453 14 6
5 7 6 9 5 5
154 30 8
8 8 6 5 5 8 6 7
726 45 6
8 6 5 7 5 8
775 2 8
2 2 2 2 2 2 2 2
129 7 9
7 7 7 5 6 5 7 7 7
419 28 8
9 7 6 6 8 7 7 8
148 40 8
9 7 8 8 7 8 6 8
584 89 6
9 9 8 8 5 8
277 23 7
7 7 5 9 5 7 8
519 70 7
8 5 6 6 8 9 6
501 20 5
8 7 9 9 8
760 16 6
7 7 5 9 9 5
129 77 5
7 6 6 6 5
