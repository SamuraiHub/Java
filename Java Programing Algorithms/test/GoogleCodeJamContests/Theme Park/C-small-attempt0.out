Case #1: 20
Case #2: 0
Case #3: 0
Case #4: 0
Case #5: 0
Case #6: 0
Case #7: 0
Case #8: 0
Case #9: 0
Case #10: 272
Case #11: 101
Case #12: 450
Case #13: 361
Case #14: 0
Case #15: 614
Case #16: 459
Case #17: 0
Case #18: 0
Case #19: 0
Case #20: 0
Case #21: 61
Case #22: 714
Case #23: 468
Case #24: 230
Case #25: 0
Case #26: 0
Case #27: 0
Case #28: 0
Case #29: 0
Case #30: 189
Case #31: 1720
Case #32: 1808
Case #33: 285
Case #34: 1568
Case #35: 85
Case #36: 498
Case #37: 2600
Case #38: 571
Case #39: 1459
Case #40: 806
Case #41: 150
Case #42: 183
Case #43: 468
Case #44: 1783
Case #45: 3192
Case #46: 1579
Case #47: 760
Case #48: 16
Case #49: 700
Case #50: 667
